### PR TITLE
Multilingual rn

### PR DESCRIPTION
**Pull request by ksyeo1010**
same as rhino, will release then update demo after approval

### CODE REVIEW

File: sdk/react-native/src/picovoice_manager.tsx [hunk 1]
```diff
@@ -197,6 +197,13 @@ class PicovoiceManager {
     this._picovoice = undefined;
     return this._voiceProcessor?.stop();
   }
+
+  /**
+   * @returns the Rhino context source YAML
```
Comment (laves) [inline sdk/react-native/src/picovoice_manager.tsx:202]: Add a note that this can only be obtained after a call to `start()`

**Review by laves — CHANGES_REQUESTED** [review]
after the small doc change, you can release the package

File: demo/react-native/App.tsx [hunk 2]
```diff
@@ -15,10 +23,15 @@ type State = {
...
+  contextInfo?: string;
+  showContextInfo: boolean;
 };
 
+const marginOffset = Platform.OS === 'ios' ? 30 : 0;
+
 export default class App extends Component<Props, State> {
-  readonly _accessKey = '${YOUR_ACCESS_KEY_HERE}'; // AccessKey obtained from Picovoice Console (https://console.picovoice.ai/)
+  readonly _accessKey =
+    'Tw4jothrMMLyRYQ793yD/XF3DeithcbeNVsYlNN0Dc1vY26suWNOkg=='; // AccessKey obtained from Picovoice Console (https://console.picovoice.ai/)
```
Comment (laves) [inline demo/react-native/App.tsx:34]: can you erase this from the history with a squash?

**Review by laves — CHANGES_REQUESTED** [review]

**Review by laves — CHANGES_REQUESTED** [review]
Context info shows: `this._picovoiceManager?.contextInfo`. I think you forgot to use backticks around it

**Comment by ksyeo1010** [comment]
> Context info shows: `this._picovoiceManager?.contextInfo`. I think you forgot to use backticks around it

oh I just released so that was there for placeholder temporarily

File: demo/react-native/App.tsx [hunk 3]
```diff
@@ -212,33 +249,52 @@ export default class App extends Component<Props, State> {
...
-            <Text style={styles.buttonText}>{this.state.buttonText}</Text>
-          </TouchableOpacity>
-        </View>
+        <Modal
+          animationType="slide"
+          transparent={true}
+          visible={this.state.showContextInfo}>
+          <View style={styles.modalView}>
+            <ScrollView style={{ flex: 0.95, marginBottom: 10 }}>
+              <Text>{this._picovoiceManager?.contextInfo}</Text>
```
Comment (laves) [inline demo/react-native/App.tsx:282]: I think you meant to use the `contextInfo` from your state. Using this I can't see the context info unless the app is in listening mode.

**Review by laves — CHANGES_REQUESTED** [review]

**Review by laves — APPROVED** [review]

**Pull request merged**